source/declarators.tex

```diff
@@ -2388,11 +2388,11 @@
 \indextext{initialization|(}
 
 \pnum
-A declarator can specify an initial value for the
+An \grammarterm{init-declarator} can specify an initial value for the
```

Comment:
Agreed. [dcl.init] started out at explaining the syntax for initialization, and is only ever so slowly morphing into a general description of all kinds of initialization, where the invoking syntax is secondary. The introductory paragraph follows that anti-pattern quite nicely.